pulp_ansible/app/migrations/0004_add_fulltext_search_indexes.py

```diff
@@ -80,6 +80,11 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.RenameField(
+            model_name='tag',
+            old_name='_id',
+            new_name='pulp_id',
+        ),
```

Comment:
Why did you update this migration instead of adding this to 0013_pulp_fields.py?

Reply:
because of this line: https://github.com/fabricio-aguiar/pulp_ansible/blob/5454and5457/pulp_ansible/app/migrations/0004_add_fulltext_search_indexes.py#L21